subsystems/indexersubsystem.py

```diff
@@ -23,39 +25,142 @@ def asString(self) -> str:
+                constants.kIndexerMotorPGain,
+                constants.kConfigurationTimeoutLimit,
+            ),
+        ):
+            return
+        if not ctreCheckError(
+            "config_kI",
+            self.indexerMotor.config_kI(
+                constants.kIndexerMotorPIDSlot,
+                constants.kIndexerMotorIGain,
+                constants.kConfigurationTimeoutLimit,
+            ),
+        ):
+            return
+        if not ctreCheckError(
+            "config_kD",
+            self.indexerMotor.config_kD(
+                constants.kIndexerMotorPIDSlot,
+                constants.kIndexerMotorDGain,
+                constants.kConfigurationTimeoutLimit,
+            ),
+        ):
+            return
+
+        if not ctreCheckError(
+            "configReverseLimitSwitchSource",
+            self.indexerMotor.configReverseLimitSwitchSource(
+                LimitSwitchSource.Deactivated,
+                LimitSwitchNormal.Disabled,
+                constants.kConfigurationTimeoutLimit,
+            ),
+        ):
+            return
+        print(f"{constants.kIndexerMotorName} Falcon Initialization Complete")
+
+        # STAGING
+        self.stagingMotor = createMotor(
             constants.kStagingMotorId,
         )
-        self.indexerSensor = LimitSwitch(
-            self.stagingMotor,
-            constants.kForwardSensorIndexer,
-            constants.kSimIndexerSensorId,
-        )
-        self.stagingSensor = LimitSwitch(
-            self.stagingMotor,
-            constants.kForwardSensorStaging,
-            constants.kSimStagingSensorId,
-        )
+        print(f"Initializing Falcon: {constants.kStagingMotorName}")
+        if not ctreCheckError(
+            "configFactoryDefault",
+            self.stagingMotor.configFactoryDefault(
+                constants.kConfigurationTimeoutLimit
+            ),
+        ):
+            return
+        if not ctreCheckError(
+            "config_kP",
+            self.stagingMotor.config_kP(
+                constants.kStagingMotorPIDSlot,
+                constants.kStagingMotorPGain,
+                constants.kConfigurationTimeoutLimit,
+            ),
+        ):
+            return
+        if not ctreCheckError(
+            "config_kI",
+            self.stagingMotor.config_kI(
+                constants.kStagingMotorPIDSlot,
+                constants.kStagingMotorIGain,
+                constants.kConfigurationTimeoutLimit,
+            ),
+        ):
+            return
+        if not ctreCheckError(
+            "config_kD",
+            self.stagingMotor.config_kD(
+                constants.kStagingMotorPIDSlot,
+                constants.kStagingMotorDGain,
+                constants.kConfigurationTimeoutLimit,
+            ),
+        ):
+            return
+        if not ctreCheckError(
+            "configForwardLimitSwitchSource",
+            self.indexerMotor.configForwardLimitSwitchSource(
```

Comment:
staging motor

Reply:
You have sharp eyes

Reply:
too many errors caused by bad config inputs...